Q12.py

```diff
@@ -0,0 +1,45 @@
+class BankAccount:
+    def __init__(self, acc_name, acc_number, balance, pin):
+        self.acc_name = acc_name
+        self.acc_number=acc_number
+        self.balance=balance
```

Comment:
There should be one space before and after `=`, for the first, you have speicified but for others you didn't specified a space.